htdocs/.htaccess

```diff
@@ -18,7 +18,8 @@ AddOutputFilterByType DEFLATE text/html text/plain text/xml text/css application
 
 <FilesMatch "styles\.json$">
 	ExpiresActive on
-	ExpiresDefault "access plus 1 hour"
+	ExpiresDefault "access plus 1 second"
+	Header set Cache-Control "must-revalidate"
```

Comment:
I don't think we actually need or want `must-revalidate`. It should be enough to just set `access plus 1 minute`, and browsers won't revalidate for a minute. After a minute is up, they'll make conditional requests.

Should use the same for .html below.